### PR TITLE
Add a RunKit example to Aphrodite

### LINKED ISSUE #140: Add a RunKit example
I noticed that there's a way to add a default example to the RunKit (formerly TonicDev?) example that for a repository: https://discuss.runkit.com/t/customizing-your-npm-packages-page-on-tonic/23

It would be cool if we added one of these for Aphrodite! I came up with a server-side rendering example that we could use: https://runkit.com/xymostech/aphrodite-server-side-rendering-example I tried to make a non-server-side-rendering example but it required jsdom and a `setTimeout` which was pretty ugly. 

Thoughts?

### PR DESCRIPTION
**Pull request by xymostech**
Summary: RunKit (previously TonicDev) lets you add a custom example on
the package page linked to from npmjs.com:
https://discuss.runkit.com/t/customizing-your-npm-packages-page-on-tonic/23

This adds such an example!

Closes #140

Test Plan:
 - Publish this as the new package, `aphrodite-test`
 - See that the runkit integration works :)

### CODE REVIEW

**Comment by xymostech** [comment]
https://runkit.com/npm/aphrodite-test

File: examples/runkit.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Aphrodite server-side rendering example
+
+// Make a component that generates some styles and returns some HTML.
+function render() {
+    const {StyleSheet, css} = require("aphrodite/no-important");
```
Comment (jlfwong) [inline examples/runkit.js]: Seems weird to use the non-default option in the example

Comment (xymostech) [inline examples/runkit.js]: :P If a console statement is too long in RunKit, it adds an ellipse, which I thought was annoying, so I went with this to fix that. I'm happy to undo that though!

image

**Comment by xymostech** [comment]
> Did you mean to check in dist/aphrodite.js? Looks like those changes have nothing to do with this review.

updated the pr, I had a git pre-commit hook which automatically built the files from back when we always committed built files. I removed it now :P

File: examples/runkit.js (hunk 1 above)
Comment (jlfwong) [inline examples/runkit.js]: Hehe, sad, but okay!

**Comment by jlfwong** [comment]
LGTM!

**Comment by xymostech** [comment]
Can you accept the review? I can push it anyway but I want to see what it looks like when you do :)

**Comment by xymostech** [comment]
Also, I brought this up a bit ago but now that we have new things to land, I'll bring it up again: Do we want to move to version 1.0 next time we release? Semver says

> If your software is being used in production, it should probably already be 1.0.0.

and Aphrodite is very in production.

**Comment by kentcdodds** [comment]
Yes please :)

**Comment by jlfwong** [comment]
@xymostech Was going to suggest we make 1.0.0 the version with plugins, though perhaps it's weird to do the bump to 1.0.0 when introducing a somewhat experimental feature? I don't have strong feels, and would be happy for us to bump immediately if you think that's more appropriate.

**Comment by xymostech** [comment]
Oh yeah, we can do it after that lands. Just making sure you're on board with doing it sometime soon :)

**Pull request merged**